elements/lisk-p2p/src/peer/base.ts

```diff
@@ -136,12 +138,17 @@ export class Peer extends EventEmitter {
 			this._resetProductivity();
 		}, DEFAULT_PRODUCTIVITY_RESET_INTERVAL);
 		this._serverNodeInfo = peerConfig.serverNodeInfo;
+		this._protocolRCPCounter = 0;
+		this._protocolRCPEvents = new Set([
+			REMOTE_EVENT_RPC_GET_NODE_INFO,
```

Comment:
move this to constants as `P2P RPC EVENTS` and use it here, so that in the future if we have new  RPC events, its easy to modify at one place.